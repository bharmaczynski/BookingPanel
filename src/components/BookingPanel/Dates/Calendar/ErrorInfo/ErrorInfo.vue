<template functional>
    <div class="error-info">
        <p>Select other days</p>
        <span>Selected date includes unavailable days</span>
    </div>
</template>
<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

@Component
export default class ErrorInfo extends Vue {}
</script>
<style lang="scss" scoped>
.error-info {
    padding: 10px 10px 20px;
    color: #ca0b00;

    p {
        text-transform: uppercase;
        margin-bottom: 5px;
    }
}
</style>
