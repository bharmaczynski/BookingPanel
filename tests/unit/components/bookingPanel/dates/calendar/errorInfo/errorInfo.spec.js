import { shallowMount } from '@vue/test-utils';
import ErrorInfo from '@/components/bookingPanel/dates/calendar/errorInfo/ErrorInfo.vue'

describe('ErrorInfo', () => {
    it('should match initial snapshot', () => {
        const wrapper = shallowMount(ErrorInfo);

        expect(wrapper.html()).toMatchSnapshot();
    });
});
