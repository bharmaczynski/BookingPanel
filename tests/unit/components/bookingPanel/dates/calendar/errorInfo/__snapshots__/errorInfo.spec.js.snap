// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorInfo should match initial snapshot 1`] = `
<div class="error-info">
  <p>Select other days</p> <span>Selected date includes unavailable days</span>
</div>
`;
